feature/security/gnsi/certz/client_certificates/README.md

```diff
@@ -0,0 +1,109 @@
+# X.X: gNSI Client Certificate Tests
+
+## Summary
+
+Clients must be able to validate a remote server's TLS certificate
+and present a valid client certificate to that server in order to provide
+identification information. The client certificate should have a
+SPIFFE Idenitifier embedded in it to be used as the identifier of
+the client to the server.
+
+## Baseline Setup
+
+### Input Args
+
+* the set of certificate testdata generated with the mk_cas.sh
+   script in featureprofiles/feature/security/certificate-authorties/
+
+### DUT service setup
+
+Configure the DUT to enable the following services (that are using gRPC) are
+up and user mTLS for authentication:
+
+* gNMI
+* gNOI
+* gNSI
+* gRIBI
+
+Be prepared to load the relevant trust_bundle.pem file for each test
+Certificate Authority(CA) under test on the DUT. Each CA has a RSA and ECDSA
+form, both must be tested.
+
+## Tests
+
+### Certz-1.0
+
+Perform these positive tests:
+
+Test that client certificates from a set of one CA are able to be validated and
+used for authentication to a device when used by a client connecting to each
+gRPC service.
+
+Perform this for both RSA and ECDSA signed CA bundles and
+certificates.
+Perform this for the permutations of 1, 2, 10, 1000 CA
+trust_bundle configurations: (## indicates the 1, 2, 10, 1000 CA testdata)
+
+   1) Load the correct key-type trust bundle onto the device and client system:
+        ca-##/trust_bundle_##_rsa.pem
+        ca-##/trust_bundle_##_ecdsa.pem
+
+   2) Load the correct key-type client certificate into the test gRPC client:
+        ca-##/client-rsa-key.pem
+        ca-##/client-rsa-cert.pem
+        ca-##/client-ecdsa-key.pem
+        ca-##/client-ecdsa-cert.pem
+
+   3) Load the correct key-type server certificate into the services on the DUT:
```

Comment:
Mention via what Certz services.

Reply:
This test is really just: 
  "Can you load certificates and use those to connect to the device/service"
  
The aim here is to see that the client can use one of several flavors of valid certificate(s) and trust_bundles. That the connection to the server/device works as expected provided the type/bundle/etc match between client/server.

There's a server_certificate_rotation test (not this one) that should be the 'how to rotate properly' test cases.